caixa/caixa_test.go

```diff
@@ -308,3 +308,55 @@ func TestCaixaValidateInterestWithNilFees(t *testing.T) {
 
 	assert.Nil(t, caixaValidateInterest(input))
 }
+
+func TestTemplateRequestCaixa_WhenRequestWithSacadorAvalistaIsCNPJ_ParseSuccessful(t *testing.T) {
+	f := flow.NewFlow()
+	input := newStubBoletoRequestCaixa().WithRecipientName("Recipient Test Name").WithRecipientDocumentType("CNPJ").Build()
+
+	b := fmt.Sprintf("%v", f.From("message://?source=inline", input, getRequestCaixa(), tmpl.GetFuncMaps()).GetBody())
+	no_content := test.GetNode(b, "SACADOR_AVALISTA")
+
+	assert.Contains(t, no_content, "<RAZAO_SOCIAL>Recipient Test Name</RAZAO_SOCIAL>", "Erro no mapeamento do campo Recipient")
+}
+
+func TestTemplateRequestCaixa_WhenRequestWithSacadorAvalistaIsCPF_ParseSuccessful(t *testing.T) {
+	f := flow.NewFlow()
+	input := newStubBoletoRequestCaixa().WithRecipientName("Recipient Test Name").WithRecipientDocumentType("CPF").Build()
+
+	b := fmt.Sprintf("%v", f.From("message://?source=inline", input, getRequestCaixa(), tmpl.GetFuncMaps()).GetBody())
+	node_content := test.GetNode(b, "SACADOR_AVALISTA")
```

Comment:
Aqui acho que ficaria melhor com o padrão de variáveis Golang 😀

```suggestion
	nodeContent := test.GetNode(b, "SACADOR_AVALISTA")
```